homeassistant/components/recorder/history.py

```diff
@@ -856,7 +856,7 @@ def _sorted_states_to_dict(
     """
     schema_version = _schema_version(hass)
     _process_timestamp: Callable[[datetime], float | str]
-    field_map: dict[str, int]
+    field_map: dict[str, int] = {}
```

Comment:
```suggestion
    field_map: dict[str, int] | None = None
```

Reply:
Replaced this line per https://github.com/home-assistant/core/pull/88529#issuecomment-1438065409